python/gigl/src/common/types/pb_wrappers/gbml_config.py

```
@@ -46,37 +46,43 @@ class GbmlConfigPbWrapper:
 
     gbml_config_pb: gbml_config_pb2.GbmlConfig
 
-    _dataset_metadata_pb_wrapper: DatasetMetadataPbWrapper = field(init=False)
+    _dataset_metadata_pb_wrapper: Optional[DatasetMetadataPbWrapper] = field(init=False)
     _graph_metadata_pb_wrapper: GraphMetadataPbWrapper = field(init=False)
     _preprocessed_metadata_pb_wrapper: PreprocessedMetadataPbWrapper = field(init=False)
-    _flattened_graph_metadata_pb_wrapper: FlattenedGraphMetadataPbWrapper = field(
-        init=False
-    )
+    _flattened_graph_metadata_pb_wrapper: Optional[
+        FlattenedGraphMetadataPbWrapper
+    ] = field(init=False)
     _task_metadata_pb_wrapper: TaskMetadataPbWrapper = field(init=False)
     _trained_model_metadata_pb_wrapper: TrainedModelMetadataPbWrapper = field(
         init=False
     )
-    _subgraph_sampling_strategy_pb_wrapper: SubgraphSamplingStrategyPbWrapper = field(
-        init=False
-    )
+    _subgraph_sampling_strategy_pb_wrapper: Optional[
```

This should have always been `Optional`, not sure why mypy didn't catch this but have adjusted this now

---

should we have this throw when not set too, instead of being optional?